### PR TITLE
feat: Pinch to zoom

### LINKED ISSUE #469: Pinch to zoom in and drag to pan on an ipad
## Is your feature request related to a problem? Please describe

When using an ipad, we want users to be able to  zoom in and out by pinching. And also drag to pan

### PR DESCRIPTION
**Pull request by ChrisBaidoo**
## Description

Allow users with touch device to be able to pinch to zoom and also pan. 

closes #469 

## Checklist:

Put an `x` in the boxes that apply to this pull request (you can also fill these out after opening the pull request). If you're unsure about any of these, don't hesitate to leave a comment on this pull request!

- [x] I have read the gliff.ai Contribution Guide.
- [x] I have requested to **pull a branch** and not from main.
- [ ] I have checked all commit message styles match the requested structure.
- [ ] My code follows the style guidelines of this project.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have performed a self-review of my own code.
- [ ] I have assigned **3 or less** reviewers.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] My changes generate no new warnings.
- [ ] I have made corresponding changes to the documentation.
- [ ] New database changes have been committed.
- [ ] If appropriate, I have bumped any version numbers.

### CODE REVIEW

**Review by joshuajames-smith — CHANGES_REQUESTED** [review]
If any annotation tool is active the two finger gesture to zoom and pan creates an annotation. This is unexpected? Undo would solve this without needing to delete the whole annotation layer but a user will expect to do this gesture and not make any annotations. Is there anyway to make the zoom and pan motion visually smoother?

Also the minimap has disappeared?

https://user-images.githubusercontent.com/83074763/165727106-cdbe8980-6741-4732-b0ec-db60b6e2d9be.mov

**Comment by philipjackson** [comment]
> If any annotation tool is active the two finger gesture to zoom and pan creates an annotation. This is unexpected? Undo would solve this without needing to delete the whole annotation layer but a user will expect to do this gesture and not make any annotations. Is there anyway to make the zoom and pan motion visually smoother?

So I _thought_ I'd fixed this already; turns out I only fixed it for the general case in which the two fingers touch the screen at different times. In the special case where they touch the screen at the same time, it creates an annotation. You should be able to replicate this - if you have some delay between the two touches then whatever annotation you've drawn with the first touch should be aborted when the second finger goes down. But it still draws when you release if you apply both fingers at the same time. I'll see what I can do about it.

I do also notice the panning kind of sucks (now that I realised I can test it on my iPhone via the azure static link). I'll have a look at that too.

**Review by joshuajames-smith — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     "@types/utif": "^3.0.1",
     "@typescript-eslint/eslint-plugin": "^4.19.0",
     "@typescript-eslint/parser": "^4.18.0",
-    "chromedriver": "^99.0.0",
+    "chromedriver": "^100.0.0",
```
Comment (ChasNelson1990) [inline package.json:49]: Do we actually need this dependency?

**Review by ChasNelson1990 — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (philipjackson) [reply package.json:49]: We do need it, and it's nowhere to be found in jest-browserstaci-automate's package.json.

Comment (cooper667) [reply package.json:49]: It's probably best listed as an optional dep (although we don't list it on Manage at all) as you'd only need it if running the visual tests locally?

**Review by cooper667 — APPROVED** [review]

**Pull request merged**